core/src/Kokkos_CopyViews.hpp

```diff
@@ -871,381 +706,55 @@ struct ViewRemap<DstType, SrcType, ExecSpace, 1> {
-                                       ext4, ext5, Kokkos::ALL);
-        view_copy(exec_space..., common_subview.dst_sub,
-                  common_subview.src_sub);
-      } else {
-        p_type ext1(0, std::min(dst.extent(1), src.extent(1)));
-        p_type ext2(0, std::min(dst.extent(2), src.extent(2)));
-        p_type ext3(0, std::min(dst.extent(3), src.extent(3)));
-        p_type ext4(0, std::min(dst.extent(4), src.extent(4)));
-        p_type ext5(0, std::min(dst.extent(5), src.extent(5)));
-        p_type ext6(0, std::min(dst.extent(6), src.extent(6)));
-        using sv_adapter_type =
-            CommonSubview<DstType, SrcType, 7, Kokkos::ALL_t, p_type, p_type,
-                          p_type, p_type, p_type, p_type>;
-        sv_adapter_type common_subview(dst, src, Kokkos::ALL, ext1, ext2, ext3,
-                                       ext4, ext5, ext6);
-        view_copy(exec_space..., common_subview.dst_sub,
-                  common_subview.src_sub);
-      }
-    } else {
-      if (dst.extent(6) == src.extent(6)) {
-        p_type ext0(0, std::min(dst.extent(0), src.extent(0)));
-        p_type ext1(0, std::min(dst.extent(1), src.extent(1)));
-        p_type ext2(0, std::min(dst.extent(2), src.extent(2)));
-        p_type ext3(0, std::min(dst.extent(3), src.extent(3)));
-        p_type ext4(0, std::min(dst.extent(4), src.extent(4)));
-        p_type ext5(0, std::min(dst.extent(5), src.extent(5)));
-        using sv_adapter_type =
-            CommonSubview<DstType, SrcType, 7, p_type, p_type, p_type, p_type,
-                          p_type, p_type, Kokkos::ALL_t>;
-        sv_adapter_type common_subview(dst, src, ext0, ext1, ext2, ext3, ext4,
-                                       ext5, Kokkos::ALL);
-        view_copy(exec_space..., common_subview.dst_sub,
-                  common_subview.src_sub);
-      } else {
-        p_type ext0(0, std::min(dst.extent(0), src.extent(0)));
-        p_type ext1(0, std::min(dst.extent(1), src.extent(1)));
-        p_type ext2(0, std::min(dst.extent(2), src.extent(2)));
-        p_type ext3(0, std::min(dst.extent(3), src.extent(3)));
-        p_type ext4(0, std::min(dst.extent(4), src.extent(4)));
-        p_type ext5(0, std::min(dst.extent(5), src.extent(5)));
-        p_type ext6(0, std::min(dst.extent(6), src.extent(6)));
-        using sv_adapter_type =
-            CommonSubview<DstType, SrcType, 7, p_type, p_type, p_type, p_type,
-                          p_type, p_type, p_type>;
-        sv_adapter_type common_subview(dst, src, ext0, ext1, ext2, ext3, ext4,
-                                       ext5, ext6);
-        view_copy(exec_space..., common_subview.dst_sub,
-                  common_subview.src_sub);
-      }
-    }
-  }
-};
-
-template <class DstType, class SrcType, class ExecSpace>
-struct ViewRemap<DstType, SrcType, ExecSpace, 8> {
+template <class DstType, class SrcType, class ExecSpace, int Rank>
```

Comment:
Seeing that this PR generalized and removed all `ViewRemap` specializations, it looks like `ExecSpace` template parameter is no longer used anywhere for this struct. Should this template parameter be deprecated/removed?